test/integration/analysis.js

```diff
@@ -7,6 +7,8 @@ var Analysis = require('../../lib/analysis');
 var BatchClient = require('../../lib/postgresql/batch-client');
 
 var testConfig = require('../test-config');
+var testHelper = require('../helper');
+var DataObservatoryMultipleMeasures = require('../../lib/node/nodes/data-observatory-multiple-measures');
```

Comment:
As this is a generic behaviour, you could create an ad-hoc analysis node just for testing purposes. Something in the line of https://github.com/CartoDB/camshaft/blob/4ba3e104ef7183e938c5b5f58f24e2247292abdc/test/unit/node-creation.js#L64-L91.